spec/enumerables_spec.rb

```diff
@@ -0,0 +1,234 @@
+require './enumerables.rb'
```

Comment:
```suggestion
require '../enumerables.rb'
```
- A period was missing in this file reference. kindly make sure you add that. Thank you